scan/src/Index.re

```diff
@@ -10,7 +10,9 @@ TimeAgos.setMomentRelativeTimeThreshold();
 
 ReactDOMRe.render(
   <ApolloClient>
-    <GlobalContext> <AccountContext> <App /> </AccountContext> </GlobalContext>
+    <GlobalContext>
+      <AccountContext> <ModalContext> <App /> </ModalContext> </AccountContext>
```

Comment:
Think carefully about that order. Does `AccountContext` need access to `ModalContext`? What about the other way around?